pkg/connector/inet/inet.go

```diff
@@ -104,10 +108,8 @@ func SendFleetAPICommand(ctx context.Context, client *http.Client, userAgent, au
 	switch result.StatusCode {
 	case http.StatusOK:
 		return body, nil
-	case http.StatusUnprocessableEntity:
-		if bytes.Contains(body, []byte("vehicle does not support signed commands")) {
-			return nil, protocol.ErrProtocolNotSupported
-		}
+	case http.StatusUnprocessableEntity: // HTTP: 422 on commands endpoint means protocol is not supported (fallback to regular commands)
```

Comment:
Have you verified that this is the only case where we return 422?

Reply:
Yeah, we only send 422 in the case of unsupported command errors on the signed_command endpoint